staging/src/k8s.io/client-go/INSTALL.md

```diff
@@ -143,6 +143,48 @@ After modifying, run `glide up -v` again to re-populate your /vendor directory.
 Optionally, Glide users can also use [`glide-vc`](https://github.com/sgotti/glide-vc)
 after running `glide up -v` to remove unused files from /vendor.
 
+### Go modules
```

Comment:
what will change when we publish go.mod?

Should me mention that this is temporary below?

Reply:
compare with https://github.com/liggitt/kubernetes/blob/go-mod-cleanup/staging/src/k8s.io/client-go/INSTALL.md#go-modules

I don't find "coming soon" things in documentation very helpful... as a user, I want to know what works now and other things are just noise